src/Ui/Table/Component/Header/Header.php

```diff
@@ -57,13 +57,13 @@ class Header implements HeaderInterface
      */
     public function getQueryString()
     {
-        $query = $_GET;
+        $query = app('request')->query();
 
         $builder   = $this->getBuilder();
         $direction = $this->getDirection('asc');
 
         array_set($query, $builder->getTableOption('prefix') . 'order_by', $this->getSortColumn());
-        array_set($query, $builder->getTableOption('prefix') . 'sort', $direction == 'asc' ? 'desc' : 'asc');
+        array_set($query, $builder->getTableOption('prefix') . 'sort', $direction === 'asc' ? 'asc' : 'desc');
```

Comment:
Actually, the native methods are much faster, as it was shown by a test.

Reply:
@RyanThompson I've addressed what I think might be a bug here...Otherwise was this method intentionally inverting the sort direction?